buildroot/share/PlatformIO/variants/MARLIN_ANET_ET4/variant.h

```diff
@@ -0,0 +1,385 @@
+//     #define PA4  A14
+//-------------------------------------------------------------------------------------------
+
+
+// Below ADC and PWM definitions already done in the core
+// Could be redefined here if needed
+// ADC resolution is 10 bits
+//#define ADC_RESOLUTION          10
+
+// PWM resolution
+//#define PWM_RESOLUTION          8
+//#define PWM_FREQUENCY           1000
+//#define PWM_MAX_DUTY_CYCLE      255
+
+// On-board LED pin number
+#define LED_BUILTIN             PD12
+#define LED_GREEN               LED_BUILTIN
+
+// On-board user button
+//#define USER_BTN
+
+// Below SPI and I2C definitions already done in the core
+// Could be redefined here if differs from the default one
+// SPI Definitions
+#define PIN_SPI_SS                 8
+#define PIN_SPI_MOSI              11
+#define PIN_SPI_MISO              10
+#define PIN_SPI_SCK                9
+
+// I2C Definitions
+#define PIN_WIRE_SDA               7
+#define PIN_WIRE_SCL               6
+
+// I2C timing definitions (optional), avoid time spent to compute if defined
+// * I2C_TIMING_SM for Standard Mode (100kHz)
+// * I2C_TIMING_FM for Fast Mode (400kHz)
+// * I2C_TIMING_FMP for Fast Mode Plus (1000kHz)
+//#define I2C_TIMING_SM           0x00000000
+//#define I2C_TIMING_FM           0x00000000
+//#define I2C_TIMING_FMP          0x00000000
+
+// Timer Definitions (optional)
+// Use TIM6/TIM7 when possible as servo and tone don't need GPIO output pin
+#define TIMER_TONE              TIM6
+#define TIMER_SERIAL            TIM7
+
+// Do not use basic timer: OC is required
+#define TIMER_SERVO             TIM2  //TODO: advanced-control timers don't work
+
+// UART Definitions
+// Define here Serial instance number to map on Serial generic name
+#define SERIAL_UART_INSTANCE    1 //ex: 2 for Serial2 (USART2)
+// DEBUG_UART could be redefined to print on another instance than 'Serial'
+//#define DEBUG_UART              ((USART_TypeDef *) U(S)ARTX) // ex: USART3
+// DEBUG_UART baudrate, default: 9600 if not defined
+//#define DEBUG_UART_BAUDRATE     x
+// DEBUG_UART Tx pin name, default: the first one found in PinMap_UART_TX for DEBUG_UART
+//#define DEBUG_PINNAME_TX        PX_n // PinName used for TX
+
+// Default pin used for 'Serial' instance (ex: ST-Link)
+// Mandatory for Firmata
+#define PIN_SERIAL_RX           PA10
+#define PIN_SERIAL_TX           PA9
+
+// Optional PIN_SERIALn_RX and PIN_SERIALn_TX where 'n' is the U(S)ART number
+// Used when user instanciate a hardware Serial using its peripheral name.
+// Example: HardwareSerial mySerial(USART3);
+// will use PIN_SERIAL3_RX and PIN_SERIAL3_TX if defined.
+//#define PIN_SERIALn_RX          x // For U(S)ARTn RX
+//#define PIN_SERIALn_TX          x // For U(S)ARTn TX
+//#define PIN_SERIALLP1_RX        x // For LPUART1 RX
+//#define PIN_SERIALLP1_TX        x // For LPUART1 TX
+
+// SD card slot Definitions
+// SD detect signal can be defined if required
+#define SD_DETECT_PIN           PD3
```

Comment:
the variant is the right place for it?

Reply:
Probably not since it's defined in the board's pins file too.

Reply:
Since this file is going away, this will be moot.